components/button/button.css

```diff
@@ -0,0 +1,56 @@
+.btn {
+	border: none;
+	padding: 0.8rem;
+	font-size: 1.8rem;
+	font-weight: 400;
+	border-radius: 0.5rem;
+	color: #fff;
+	min-width: 12rem;
+	cursor: pointer;
+}
+
+.btn-container {
+	margin-bottom: 4rem;
+	display: flex;
+}
+
+.btn-container button {
+	margin-right: 1rem;
+}
+
+.btn-primary {
+	background-color: var(--primary-color);
+}
+
+.btn-secondary {
+	background-color: #172b4d;
+}
+
+.btn-danger {
+	background-color: var(--primary-hover);
+}
+
+.btn-link {
+	background-color: var(--primary-color);
+}
+
+.fas {
```

Comment:
Hey, great work, i have one suggestion 
- if styling a specific element here, you can nest it like `.icon-btn .fas` ,or this might happen when user imports button styles, all icons with .fas as a class (most FA icons have it) will get the styles applied.
- you can use comments to separate component code from docs code so we can review faster
If that is not the case everything else looks great.

Reply:
Done with changes Ankur. Thank you for great suggestion